### PR TITLE
Added unit tests for the find offers page

### LINKED ISSUE #103: (SP: 1) Write unit test for "Find offers" page
### Page unit test
Unit test for "**Find offers**" page
Scenaries descriptions:
- [ ] It should render page
- [ ] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-UA-1156/Client-01/blob/develop/src/pages/find-offers/FindOffers.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by holus-bolus**
Created unit tests for the Find offers page
image

### CODE REVIEW

**Review by YaroslavChuiko — APPROVED** [review]

**Review by OlhaOvsiienko — APPROVED** [review]

**Review by khytrykv — APPROVED** [review]

File: tests/unit/pages/find-offers/FindOffers.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    closeModal: mockCloseModal
+  })
+}))
+
+describe('FindOffers component', () => {
+  it('renders the Find Offers page', async () => {
+    render(
+      <ModalProvider>
+        <FindOffers />
+      </ModalProvider>
```
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx]: Use `renderWithProviders` from '~tests/test-utils'

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Mock `<PageWrapper>` component.

File: tests/unit/pages/find-offers/FindOffers.spec.jsx (hunk 1 above)
Comment (holus-bolus) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx]: @vlad-khrychov corrected the PR accordingly, unfortunately, I can't use the ~ custom src in this unit test

Comment (holus-bolus) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx]: @vlad-khrychov fixed

**Review by vlad-khrychov — APPROVED** [review]
Before merging, check if everything works fine with the new version of vitest

**Pull request merged**